### PR TITLE
Supports brightness and contrast augmentations

### PR DESCRIPTION
**Pull request by v-qjqs**
This PR implements brightness and contrast augmentations in mmcv.

### CODE REVIEW

File: mmcv/image/photometric.py [hunk 1]
```diff
@@ -166,3 +166,61 @@ def _scale_channel(im, c):
...
+    blends the source image and the degenerated image:
+
+    ``output = img * alpha + degenerated * (1 - alpha)``
+
+    Args:
+        img (ndarray): Image to be brightened.
+        alpha (float): A value controls the enhancement.
+            Factor 1.0 returns the original image, lower
+            factors mean less color (brightness, contrast,
+            etc), and higher values more.
```
Comment (xvjiarui) [inline mmcv/image/photometric.py:186]: Add default value.

File: tests/test_image/test_photometric.py [hunk 2]
```diff
@@ -135,3 +135,65 @@ def _imequalize(img):
...
+                np.random.uniform(0, 1, (1200, 1000, 3)) * 260, 0,
+                255).astype(np.uint8)
+            factor = np.random.uniform()
+            # Note the gap (less_equal 1) between PIL.ImageEnhance.Contrast
+            # and mmcv.adjust_contrast comes from the gap that converts from
+            # a color image to gray image using mmcv or PIL.
+            delta = np.absolute(
+                mmcv.adjust_contrast(img, factor) -
+                _adjust_contrast(img, factor))
+            assert np.less_equal(delta, 1).all()
```
Comment (xvjiarui) [inline tests/test_image/test_photometric.py:199]: We may use `np.allclose`.

File: mmcv/image/photometric.py [hunk 3]
```diff
@@ -166,3 +166,61 @@ def _scale_channel(im, c):
...
+    return brightened_img.astype(img.dtype)
+
+
+def adjust_contrast(img, alpha=1.):
+    """Adjust image contrast.
+
+    This function controls the contrast of an image. An
+    enhancement factor (alpha) of 0.0 gives a solid grey
+    image. A factor of 1.0 gives the original image. It
+    blends the source image and the degenerated image:
```
Comment (xvjiarui) [inline mmcv/image/photometric.py:207]: We may also elaborate on what degenerated image is.

File: mmcv/image/photometric.py [hunk 4]
```diff
@@ -166,3 +166,61 @@ def _scale_channel(im, c):
     s3 = _scale_channel(img, 2)
     equalized_img = np.stack([s1, s2, s3], axis=-1)
     return equalized_img
+
+
+def adjust_brightness(img, alpha=1.):
```
Comment (hellock) [inline mmcv/image/photometric.py:171]: We can just rename alpha to factor.

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**